.github/workflows/shorebird_ci.yml

```diff
@@ -57,5 +57,5 @@ jobs:
           java-version: "11"
 
       - name: 🐦 Run Shorebird Tests
-        run: dart test
+        run: dart test --concurrency=1
```

Comment:
This will probably make the total test process take more time, but will make it more stable, decreasing the risk of running into timeout errors

Reply:
Can we mark the offending tests as slow instead?

Reply:
I am unsure what marking a test as slow means? Is that a dart test config or something?

Either way I wonder if it is worth it, all the tests in `shorebird_tests` are kinda of integration tests, which will create a flutter project, run something in it and do assertions, so, all tests will take a while.

Reply:
I think he meant you can increase the timeout for the specific slow tests

Reply:
```suggestion
        run: dart test
```

Reply:
I guess I actually changed the concurrency because I thought we would get lock issues between the tests, but after debugging this I realized the lock was only happening because there were a small issue in the code.

I removed it